src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -172,15 +174,20 @@
         [:div.application-handling__review-header "Tila"]]
        (mapv (partial review-state-row @review-state) application-review-states)))))
 
+(defn event-caption [event]
+  (match (:event-type event)
```

Comment:
Maybe `(case (:event-type event) ...)` would be enough here? `clojure.core.match/match` version is 2.9 times as slow as `clojure.core/case`.